invenio.cfg

```diff
@@ -189,7 +189,37 @@ CELERY_BEAT_SCHEDULE = {
     "cleanup-swh-depositions": {
         "task": "invenio_swh.tasks.cleanup_depositions",
         "schedule": crontab(minute=0, hour=6), # Every day at 06:00 UTC
-    }
+    },
+    'file-checks': {
+        'task': 'invenio_files_rest.tasks.schedule_checksum_verification',
+        'schedule': timedelta(hours=1),
+        'kwargs': {
+            'batch_interval': {'hours': 1},
+            'frequency': {'days': 14},
+            'max_count': 0,
+            # Query taking into account only files with URI prefixes defined by
+            # the FILES_REST_CHECKSUM_VERIFICATION_URI_PREFIXES config variable
+            'files_query':
+                'zenodo_rdm.files.checksum_verification_files_query',
+        },
+    },
+    'hard-file-checks': {
+        'task': 'invenio_files_rest.tasks.schedule_checksum_verification',
+        'schedule': timedelta(hours=1),
+        'kwargs': {
+            'batch_interval': {'hours': 1},
+            # Manually check and calculate checksums of files biannually
+            'frequency': {'days': 180},
+            # Split batches based on total files size
+            'max_size': 0,
+            # Query taking into account only files with URI prefixes defined by
+            # the FILES_REST_CHECKSUM_VERIFICATION_URI_PREFIXES config variable
+            'files_query':
+                'zenodo_rdm.files.checksum_verification_files_query',
+            # Actual checksum calculation, instead of relying on a EOS query
+            'checksum_kwargs': {'use_default_impl': True},
```

Comment:
I don't find where `use_default_impl` is implemented, can you share how that works?

Reply:
This is copy-pasted from the original [config we had in legacy](https://github.com/zenodo/zenodo/blob/482ee72ad501cbbd7f8ce8df9b393c130d1970f7/zenodo/config.py#L240-L241). The argument is used [in the XRootD storage class](https://github.com/inveniosoftware/invenio-xrootd/blob/5d1507fc85619958ce8bd0f038bfd116143f06fb/invenio_xrootd/storage.py#L64), to basically force it to "manually" compute the MD5 checksum on our side (instead of relying on EOS).